app/views/user_training/required_trainings.haml

```diff
@@ -0,0 +1,35 @@
+%h1 Additional Training is Required
+
+%div The training(s) identified below are required to be completed prior to accessing the warehouse.
+%div Please contact an administrator to setup your account in the following TalentLMS domain(s) and ensure that the correct email address linked to your warehouse account.
+%br
+%h2 Account Information
+.well
+  %div
+    Warehouse email:
+    = current_user.email
+  %div
+    TalentLMS email:
+    = current_user.talent_lms_email
+
+%h2 Required Training Course Information
+- @configs.sort_by(&:unique_name).each do |config|
+  .well
+    %h3= config.unique_name
+    %p 
+      Domain:
+      = link_to config.domain, "https://#{config.domain}"
+    .table-responsive
+      %table.table.table-striped
+        %thead
+          %tr
+            %th Course Name
+            %th Course ID
+        %tbody
+          - required_courses_for_config = (@courses & config.courses)
+          - required_courses_for_config.each do |course|
+            %tr
+              %td= course.name
+              %td= course.courseid
```

Comment:
Can you add a "Completed" column to this with a yes/no check/x for completed/incomplete?  That way people can see the progress toward being able to access the site.

Reply:
Updated the portal to display this info in [this commit](https://github.com/greenriver/hmis-warehouse/pull/4887/commits/a820796caba836fcd7d38d854f5958b46021b677)